### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by rtthomas2fa**

### CODE REVIEW

File: api/java/Dockerfile [hunk 1]
```diff
@@ -7,7 +7,9 @@ COPY pom.xml .
 COPY src src
 COPY .env .env
 
-RUN mvn clean package
+ENV SIMULATE_MONGO true
+RUN mvn clean install 
+ENV SIMULATE_MONGO false
```
Comment (Fosol) [inline api/java/Dockerfile]: You're overwriting the above variable two lines above.

Comment (rtthomas2fa) [reply api/java/Dockerfile]: Ok, we're back to the issue of what the automated tests are supposed to be. If they are not to test the queuing and status setting functionality, what are they supposed to be doing?

Comment (Fosol) [reply api/java/Dockerfile]: unit tests are not integration tests.  The unit test should only test the details within the function, not the dependencies.  It requires mocking those dependencies.

Comment (rtthomas2fa) [reply api/java/Dockerfile]: The SIMULATE_MONGO = true implies that no real database is used. It will connect to an embedded DB. Does that clarify things?

Comment (rtthomas2fa) [reply api/java/Dockerfile]: i.e. a "mocked" database

Comment (rtthomas2fa) [reply api/java/Dockerfile]: Sorry for the confusion. I mentioned in the review working on getting the embedded DB to work. Thought you realized what it was meant for.

**Pull request merged**